docs/guide/search/elastic.md

```diff
@@ -15,6 +15,11 @@
     ## Step 1: Download and install Elasticsearch
 
     [Install Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/7.7/install-elasticsearch.html) on your server.
+    For example, use the following [Docker](https://docs.docker.com/get-started/overview/) command:
+    
+    ```
+    docker run -d --name commerce-elasticsearch -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.7.1
```

Comment:
hmm, the container name `commerce-elasticsearch` was dedicated to a commerce use case. Here we can do something like `ibexa-dxp-elasticsearch` or skip it entirely as it's arbitrary

Reply:
Fixed!